dvc/remote/ssh/connection.py

```diff
@@ -128,7 +128,17 @@ def walk_files(self, directory):
 
         self._sftp_connect()
 
-        for entry in self._sftp.listdir_attr(directory):
+        try:
+            dir_entries = self._sftp.listdir_attr(directory)
+        except IOError as exc:
+            raise DvcException(
+                'couldn\'t get the "{}" remote directory files list'.format(
```

Comment:
```suggestion
                "couldn't get the '{}' remote directory files list".format(
```

Reply:
this is how black formats it :(

Reply:
@ei-grad It doesn't format it that way for me.

Reply:
Oh, I used the double quotes near directory name. With a single quotes it works. Updated to your version :).